api/client/gro_client.py

```diff
@@ -6,17 +6,18 @@
 import getpass
 import itertools
 import functools
+import math
 import os
 import pandas
 import sys
 import unicodecsv
 from api.client import cfg, lib, Client
-
+import geopandas as gpd
```

Comment:
Shippable's failing because we don't have geopandas and shapely in our requirements.

And actually including this requirement is a big deal since geopandas doesn't work on Windows pip and we *require* users to use pip to install the client. So adding this effectively breaks all Windows support as is.

Will need to separate this out into an additional geo-utils library or something that is not installed by default, or start distributing binaries on Anaconda Cloud so conda can be our first-class Windows installer

cc @JGrinst

Reply:
yes, we'd like to avoid geopandas as a requirement... I think we can get the functionalitywe want via shapely